ecdsa.c

```diff
@@ -36,6 +36,9 @@
 #include "base58.h"
 #include "macros.h"
 #include "secp256k1.h"
+#ifdef USE_ETHEREUM
```

Comment:
This should be #if USE_ETHEREUM.

Or change options.h to not define USE_ETHEREUM/USE_KECCAK to 0 and change all #if to #ifdef.